instruqt/distributing-with-replicated/02-enabling-the-sdk/assignment.md

```diff
@@ -0,0 +1,99 @@
+---
+slug: enabling-the-sdk
+id: k2xalui3fpuw
+type: challenge
+title: Enabling the Replicated SDK
+teaser: Incorporate the SDK into your application
+notes:
+- type: text
+  contents: Intoducing the Replicated SDK into your application
+tabs:
+- title: Shell
+  type: terminal
+  hostname: shell
+- title: Manifest Editor
+  type: code
+  hostname: shell
+  path: /home/replicant
+difficulty: basic
+timelimit: 300
+---
+
+Now that we've got our environment set up, let's incorporate the
+SDK into our application. We make it easy for you to do this by
```

Comment:
Instead of `We make it easy` how about `Replicated makes it easy` - sometimes I've noticed in the lab instruction we use `we did X` to describe something that is specific to what we did in the lab setup vs. statements about Replicated in general.

Reply:
Great point!